.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -31,3 +31,7 @@ Some brief examples of release notes:
 ```release-note
 
 ```
+
+## Testing
+
+- [ ] I've read [Prometheus-Operator testing guidelines](../TESTING.MD)
```

Comment:
Suggestion: Thanos has a Verification section in their [pull request template](https://raw.githubusercontent.com/thanos-io/thanos/main/.github/PULL_REQUEST_TEMPLATE.md) which I find quite useful.

```suggestion
## Verification

<!-- How you tested it? How do you know it works? -->

Please check the [Prometheus-Operator testing guidelines](../TESTING.MD) for recommendations about automated tests.
```

Reply:
I'm not sure about the checkbox, I fear it's going to be noise for regular contributors and skipped by the others.